build-logic/src/main/kotlin/io/github/droidkaigi/confsched2023/primitive/AndroidGradleDsl.kt

```diff
@@ -62,3 +67,46 @@ fun Project.setupAndroid() {
         }
     }
 }
+
+fun Project.setupDetekt(extension: DetektExtension) {
+    extension.apply {
+        // 並列処理
```

Comment:
We are using English for comments.
Could you use English?🙇
You can use DeepL or something.

Reply:
Fixed. 67fd868